Readme.md

```diff
@@ -54,14 +54,38 @@ That's it!
 If you need to pass some options for `react-snap`, you can do this in the `package.json`, like this:
 
 ```json
-"reactSnap": {
-  "inlineCss": true
+{
```

Comment:
not sure in usefulness of this block - it takes a lot of space but doesn't communicate much information.

Reply:
Ok, take a look